### ISSUE TITLE
Cannot resolve fields from mixins

### ISSUE THREAD

**Issue by orsinium** [dropbox/sqlalchemy-stubs#75]
The plugin can't find fields from mixins or abstract models.

```python
class _UserMixin:
    some_field = s.Column(s.Integer)

class User(_UserMixin, Base):
    ...

User(some_field=1, ...)
```

Get error:

```bash
error: Unexpected column "some_field" for model "User"
```

Same error in this case:

```python
class _UserBase(Base):
    __abstract__ = True
    some_field = s.Column(s.Integer)

class User(_UserBase):
    ...

User(some_field=1, ...)
```

**Comment by ilevkivskyi** [comment]
I didn't include this in the initial implementation because I didn't think it is a widely used feature. This should be quite straightforward to implement, essentially as I understand we will need to collect columns from the whole MRO instead of just the model class body.

**Comment by orsinium** [comment]
Thank you for your reply :) I personally don't like mixins, but there it is required because we have big batch of the same fields in the different models. We could avoid this through normalization, but this normalization would force us to use much of joins. It could slow down queries building (for developers) and execution (for database).

**Comment by MrSaints** [comment]
I believe this affects polymorphic models too: https://docs.sqlalchemy.org/en/latest/orm/inheritance.html

**Comment by ilevkivskyi** [comment]
Fixed by #80
